src/common/protocol-handler/router.ts

```diff
@@ -179,7 +179,7 @@ export abstract class LensProtocolRouter extends Singleton {
     const { [EXTENSION_PUBLISHER_MATCH]: publisher, [EXTENSION_NAME_MATCH]: partialName } = match.params;
     const name = [publisher, partialName].filter(Boolean).join("/");
 
-    const extension = ExtensionLoader.getInstance().userExtensionsByName.get(name);
+    const extension = ExtensionLoader.getInstance().getInstanceByName(name);
```

Comment:
Couldn't you have just used the existing `getExtensionByName()`?

Reply:
Do you mean instead of changing its name to `getInstanceByName`?

Reply:
I changed the name because we have a difference between `InstalledExtension`'s (which use the name `extension` in the loader's method names and `LensExtension`'s (or instances) which tend to use the work `instance` in the loader's method names.

I updated this function name to be inline with that.